docs/sources/enterprise/request-security.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Request security"
+description = "Grafana Enterprise request security"
+keywords = ["grafana", "security", "enterprise"]
+weight = 110
++++
+
+# Request security
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+Request security introduces ways to limit requests from the Grafana server and focuses on requests generated by users.
+
+Example:
+- datasource metric queries
+- alert notifications
+
+This can be used to limit access to internal systems that the server Grafana runs on can access but that users of Grafana should not be able to access. This feature does not affect traffic from the Grafana users browser.
+
+Note: While request security works with backend plugins, it is possible to create a backend plugin that bypasses this security.
+
+## IP and hostname blocking
+
+It's possible to limit requests based on hostname and/or ip addresses in two ways.
+
+### Deny list
+
+Any request to a host or IP on the deny-list will be blocked.
+
+### Allow list
+
+If there are at least one host or IP on the allow-list any request to hosts or IP:s not on the allow-list will be blocked.
```

Comment:
```suggestion
If there is at least one hostname or IP address on the allow-list, any request to a hostname or IP address that is not on the allow-list is blocked.
```

This seems confusing to me. Can we write two sentences:

"Any request to a hostname or IP address that is only on the allow-list is allowed."

"Any request to a hostname or IP address on the allow-list that is also on the deny-list is denied."

Reply:
No, that's no quite what I'm trying to say. How about this?

```markdown
Any request to a hostname or IP not on the allow list is blocked. The allow list is only activated if there is at least one hostname or IP in it.
```

Reply:
The allow list is only activated if there is at least one hostname or IP address on it. Furthermore, any request to a hostname or IP address that is not on the allow list is denied.

Reply:
Updated my suggestion. How about now?

Reply:
I think that works.